lib/commands/config/advanced.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
         description: 'Blog URL with protocol E.g. http://loveghost.com',
         validate: value => validator.isURL(value, {require_protocol: true}) ||
             'Invalid URL. Your URL should include a protocol, E.g. http://my-ghost-blog.com',
+        transform: value => { return value.toLowerCase(); },
```

Comment:
gonna be a nitpick - can this be `value => value.toLowerCase();` in keeping with the shorthand syntax above?